reference/extensions.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: page
+title: Extensions
+---
+
+Extensions are similar to [Plugins](../plugins), except for the ways they are not:
+
+  * extensions are typically just modules (really `ActiveSupport::Concern`s) that provide
+  additional logic and behavior to the specific class they're included in.
+    * or, they are simply additional classes wrapped up in a file you `require`.
+  * extensions do not have any configuration data
+  * extensions never instantiated during application loading, nor do they
+  provide callback hooks (beyond that from `ActiveSupport::Concern.included` )
+
+Praxis currently includes the following extensions, all under the `Praxis::Extensions`
+namespace:
+
+  * [`FieldSelection`](./field_selection): adds an enhanced version of `Attributor::FieldSelector`
```

Comment:
I started some of the high-level wiki pages for Attributor....and an initial structure.
There is a section that lists all the embedded types (I guess the FieldSelector is not there yet)
...but I think we can add a bulletpoint in that main page for it, and simply link and create a page for the type, that at least has "some" information about what it is (to begin with).
here: https://github.com/rightscale/attributor/wiki